exporter/jaeger/test/opentelemetry/exporters/jaeger/agent_exporter_test.rb

```diff
@@ -5,53 +5,57 @@
 # SPDX-License-Identifier: Apache-2.0
 require 'test_helper'
 
-describe OpenTelemetry::Exporter::Jaeger::Exporter do
-  SUCCESS = OpenTelemetry::SDK::Trace::Export::SUCCESS
-  FAILURE = OpenTelemetry::SDK::Trace::Export::FAILURE
-
+describe OpenTelemetry::Exporter::Jaeger::AgentExporter do
   describe '#initialize' do
     it 'initializes' do
-      exporter = OpenTelemetry::Exporter::Jaeger::Exporter.new(service_name: 'test', host: '127.0.0.1', port: 6831)
+      exporter = OpenTelemetry::Exporter::Jaeger::AgentExporter.new(host: '127.0.0.1', port: 6831)
       _(exporter).wont_be_nil
     end
   end
 
   describe '#export' do
-    let(:exporter) { OpenTelemetry::Exporter::Jaeger::Exporter.new(service_name: 'test', host: '127.0.0.1', port: 6831) }
+    let(:exporter) { OpenTelemetry::Exporter::Jaeger::AgentExporter.new(host: '127.0.0.1', port: 6831) }
 
     before do
       OpenTelemetry.tracer_provider = OpenTelemetry::SDK::Trace::TracerProvider.new
     end
 
+    it 'integrates with collector' do
+      skip unless ENV['TRACING_INTEGRATION_TEST']
+      span_data = create_span_data
+      result = exporter.export([span_data])
+      _(result).must_equal(OpenTelemetry::SDK::Trace::Export::SUCCESS)
+    end
```

Comment:
I've run this locally with:
```sh
TRACING_INTEGRATION_TEST=1 bundle exec rake test
```
and the collector:
```sh
./otelcol --config logging.yml
```
where `logging.yml` is:
```yaml
receivers:
  jaeger:
    protocols:
      thrift_compact:
      thrift_http:
  otlp:
    protocols:
      http:
exporters:
  logging:
extensions:
  zpages:
service:
  extensions: [zpages]
  pipelines:
    traces:
      receivers: [otlp, jaeger]
      exporters: [logging]
```

It produces the output:
```
2020-09-03T16:29:35.321-0400	INFO	loggingexporter/logging_exporter.go:102	TraceExporter	{"#spans": 1}
```